resources/json/pokemonIdAndName.json

```diff
@@ -0,0 +1,455 @@
+	},
+	"71": {
+		"name": "Victreebel"
+	},
+	"72": {
+		"name": "Tentacool"
+	},
+	"73": {
+		"name": "Tentacruel"
+	},
+	"74": {
+		"name": "Geodude"
+	},
+	"75": {
+		"name": "Graveler"
+	},
+	"76": {
+		"name": "Golem"
+	},
+	"77": {
+		"name": "Ponyta"
+	},
+	"78": {
+		"name": "Rapidash"
+	},
+	"79": {
+		"name": "Slowpoke"
+	},
+	"80": {
+		"name": "Slowbro"
+	},
+	"81": {
+		"name": "Magnemite"
+	},
+	"82": {
+		"name": "Magneton"
+	},
+	"83": {
+		"name": "Farfetchd"
+	},
+	"84": {
+		"name": "Doduo"
+	},
+	"85": {
+		"name": "Dodrio"
+	},
+	"86": {
+		"name": "Seel"
+	},
+	"87": {
+		"name": "Dewgong"
+	},
+	"88": {
+		"name": "Grimer"
+	},
+	"89": {
+		"name": "Muk"
+	},
+	"90": {
+		"name": "Shellder"
+	},
+	"91": {
+		"name": "Cloyster"
+	},
+	"92": {
+		"name": "Gastly"
+	},
+	"93": {
+		"name": "Haunter"
+	},
+	"94": {
+		"name": "Gengar"
+	},
+	"95": {
+		"name": "Onix"
+	},
+	"96": {
+		"name": "Drowzee"
+	},
+	"97": {
+		"name": "Hypno"
+	},
+	"98": {
+		"name": "Krabby"
+	},
+	"99": {
+		"name": "Kingler"
+	},
+	"100": {
+		"name": "Voltorb"
+	},
+	"101": {
+		"name": "Electrode"
+	},
+	"102": {
+		"name": "Exeggcute"
+	},
+	"103": {
+		"name": "Exeggutor"
+	},
+	"104": {
+		"name": "Cubone"
+	},
+	"105": {
+		"name": "Marowak"
+	},
+	"106": {
+		"name": "Hitmonlee"
+	},
+	"107": {
+		"name": "Hitmonchan"
+	},
+	"108": {
+		"name": "Lickitung"
+	},
+	"109": {
+		"name": "Koffing"
+	},
+	"110": {
+		"name": "Weezing"
+	},
+	"111": {
+		"name": "Rhyhorn"
+	},
+	"112": {
+		"name": "Rhydon"
+	},
+	"113": {
+		"name": "Chansey"
+	},
+	"114": {
+		"name": "Tangela"
+	},
+	"115": {
+		"name": "Kangaskhan"
+	},
+	"116": {
+		"name": "Horsea"
+	},
+	"117": {
+		"name": "Seadra"
+	},
+	"118": {
+		"name": "Goldeen"
+	},
+	"119": {
+		"name": "Seaking"
+	},
+	"120": {
+		"name": "Staryu"
+	},
+	"121": {
+		"name": "Starmie"
+	},
+	"122": {
+		"name": "Mr.Mime"
+	},
+	"123": {
+		"name": "Scyther"
+	},
+	"124": {
+		"name": "Jynx"
+	},
+	"125": {
+		"name": "Electabuzz"
+	},
+	"126": {
+		"name": "Magmar"
+	},
+	"127": {
+		"name": "Pinsir"
+	},
+	"128": {
+		"name": "Tauros"
+	},
+	"129": {
+		"name": "Magikarp"
+	},
+	"130": {
+		"name": "Gyarados"
+	},
+	"131": {
+		"name": "Lapras"
+	},
+	"132": {
+		"name": "Ditto"
+	},
+	"133": {
+		"name": "Eevee"
+	},
+	"134": {
+		"name": "Vaporeon"
+	},
+	"135": {
+		"name": "Jolteon"
+	},
+	"136": {
+		"name": "Flareon"
+	},
+	"137": {
+		"name": "Porygon"
+	},
+	"138": {
+		"name": "Omanyte"
+	},
+	"139": {
+		"name": "Omastar"
+	},
+	"140": {
+		"name": "Kabuto"
+	},
+	"141": {
+		"name": "Kabutops"
+	},
+	"142": {
+		"name": "Aerodactyl"
+	},
+	"143": {
+		"name": "Snorlax"
+	},
+	"144": {
+		"name": "Articuno"
+	},
+	"145": {
+		"name": "Zapdos"
+	},
+	"146": {
+		"name": "Moltres"
+	},
+	"147": {
+		"name": "Dratini"
+	},
+	"148": {
+		"name": "Dragonair"
+	},
+	"149": {
+		"name": "Dragonite"
+	},
+	"150": {
+		"name": "Mewtwo"
+	},
+	"151": {
+		"name": "Mew"
+	}
+}
```

Comment:
Isn't there already resources/pokemonList.json with the list of pokemon IDs and names ?